### DISCUSSION TITLE
How to contribute?

### DISCUSSION THREAD

**Discussion by nilsreichardt** [openpotato/openholidaysapi.website#30] [Q&A] [answered]
I just wanted to ask how the data (`.csv`) files are generated? By hand? Or is there a magic script that pulls the data from the sources?

**Comment by fstueber** [comment, marked as answer]
No magic. Just taking data manually from the sources: https://www.openholidaysapi.org/en/sources/

**Comment by michaelzoidl** [comment]
I just stumbled upon this after I created https://github.com/openpotato/openholidaysapi.data/issues/81. Does that mean I could contribute here and add the missing date in a CSV? I was just thinking about creating a PR - but how are the UUIIDs in the first column generated? (https://github.com/openpotato/openholidaysapi.data/blob/develop/src/es/holidays/holidays.public.csv)

Best, Michi

**Reply by fstueber** [reply]
Yes you can. For UUID/GUID you can generate via https://www.guidgenerator.com
